Doc 806 split environment variables page (#1991)
n8n-io/n8n-docs: docs/hosting/configuration/environment-variables/queue-mode.md

---
title: Queue mode environment variables
description: Environment variables to configure queue mode on your self-hosted n8n instance. 
contentType: reference
tags:
  - environment variables
hide:
  - toc
  - tags
---

# Queue mode environment variables

> You can run n8n in different modes depending on your needs, and the queue mode provides the best scalability. Refer to [Queue mode](/hosting/scaling/queue-mode/) for its configuration details.

Reviewer: ```suggestion
You can run n8n in different modes depending on your needs. Queue mode provides the best scalability. Refer to [Queue mode](/hosting/scaling/queue-mode/) for more information.
```
